src/components/PrivacyAgent.js

```diff
@@ -0,0 +1,58 @@
+class PrivacyAgent { // eslint-disable-line no-unused-vars
+    /**
+     * @param {string} nexthash - the hash to route to when the user clicked "ok"
+     * @param {HTMLElement} [el]
+     */
+    constructor(nexthash = 'continue', el) {
+        this.$el = el || this._createElement(nexthash);
```

Comment:
Forward the el to the _createElement function to use as optional uninitialized root element, just like in `RecoverWordsInput` component.